zanata-war/src/main/java/org/zanata/action/ProjectAction.java

```diff
@@ -60,12 +60,10 @@ public DataModel getProjectPagedListDataModel() {
     }
 
     public boolean isShowObsolete() {
-        return showObsolete;
+        return false;
```

Comment:
If this always returns `false`, its best to remove this method and update the caller. Don't see the point of having it.